Paccy10/ampersand_app_api
# Freature(motorcycle): create motorcycle

**What does this PR do?**

- Create a motorcycle

**Description of Task to be completed?**

- Register a new motorcycle

**How should this be manually tested?**

- Checkout to branch `ft-create-motocycle`
- Run the app with `flask run`
- Send a `POST` request to `/motorcycles` with the required payload
- A new motorcycle record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
